src/options.js

```diff
@@ -74,7 +73,6 @@ export type RealmOptions = {
   reactOptimizeNestedFunctions?: boolean,
   stripFlow?: boolean,
   abstractValueImpliesMax?: number,
-  debuggerConfigArgs?: DebuggerConfigArguments,
```

Comment:
Had to remove `DebuggerConfigArgs` from realm to avoid import of `./types.js` to avoid flow cycle